kotlin-reference-server/src/main/kotlin/org/stellar/reference/event/processor/Sep6EventProcessor.kt

```diff
@@ -383,11 +383,21 @@ class Sep6EventProcessor(
           event.payload.transaction.id,
           missingFields,
         )
+        val existingCustomer =
```

Comment:
Can we slightly change the logic for customer?
1. request_kyc
2. verify_kyc transition into PROCESSING if id_number is provided, otherwise more fileds needed
3. spawn a coroutine that waits 3 seconds and moves status into ACCEPTED
This will simulate a typical anchor flow. 
(We can do it in a separate PR)

Reply:
That's a good idea. I'll do this in a separate PR. I would like the refactor this class first since it's becoming very difficult to understand and extend.

Reply:
Created ANCHOR-778.